docs/overview.md

```diff
@@ -581,6 +582,8 @@ Gives false-positives for:
 
 
 
+
+
```

Comment:
I think we should fix `makedocs` tool :)

Reply:
Maybe. I don't know why it did that..
Needs investigation and dedicated issue.

Reply:
Could you file an issue?

Reply:
yep